client/apps/edit/components/content2/sections/image_collection/index.coffee

```diff
@@ -126,25 +134,31 @@ module.exports = React.createClass
             },
               images.map (item, i) =>
                 if item.type is 'artwork'
-                  Artwork {
-                    key: i
-                    index: i
-                    artwork: item
-                    removeItem: @removeItem
-                    editing:  @props.editing
-                    imagesLoaded: @state.imagesLoaded
-                    dimensions: @state.dimensions
-                    article: @props.article
-                  }
+                  React.createElement(
+                    Artwork.default, {
+                      key: i
+                      index: i
+                      artwork: item
+                      removeItem: @removeItem
+                      editing:  @props.editing
+                      imagesLoaded: @state.imagesLoaded
+                      dimensions: @state.dimensions
+                      article: @props.article
+                    }
+                  )
                 else
-                  Image {
-                    index: i
-                    key: i
-                    image: item
-                    removeItem: @removeItem
-                    editing:  @props.editing
-                    dimensions: @state.dimensions
-                    imagesLoaded: @state.imagesLoaded
-                  }
+                  React.createElement(
+                    Image.default, {
```

Comment:
Kinda strange that you'd have to reference the default here. It looks like your image and artwork does export the class as default so I wonder why it needs to be referenced like this?

Reply:
Yeah have been toying with it but is the only way I've gotten the reaction components to work in coffeescript- otherwise I get an error that it received a string. In es6 components we do not need the .default to import